drivers/SmartThings/jbl/profiles/jbl.yml

```diff
@@ -0,0 +1,30 @@
+name: jbl
+components: 
+- id: main
+  capabilities:
+  - id: mediaPlayback
+    version: 1
+    config:
+      values:
+        - key: "playbackStatus.value"
```

Comment:
This means that the device will never be in the "Stopped" state, only the Playing or Paused states. Is that true?

Reply:
Yes, that's the way, JBL bar only support Play and pause, no "stopped" state.